include/manif/impl/eigen.h

```diff
@@ -193,6 +193,34 @@ Eigen::Matrix<Scalar, 3, 1> randPointInBall(Scalar radius)
   );
 }
 
+template <typename Scalar>
+Eigen::Quaternion<Scalar> randQuat()
+{
+#if EIGEN_VERSION_AT_LEAST(3,3,0)
+
+  return Eigen::Quaternion<Scalar>::UnitRandom();
```

Comment:
Long ago we discussed quat randomness. I seem to remember that Eigen implementation was perhaps non-uniform on the unit sphere. If I am right... does UnitRandom fix this?

Reply:
The implementation we used until now (which is still available at the lines below) is a literal copy/paste from `Eigen::Quaternion<Scalar>::UnitRandom`. So this technically does not change anything.